Added information about policy content/read for Reviewers workflow (#1840)
ezsystems/developer-documentation: docs/guide/workflow/workflow.md

When moving a Content item through a transition, the user can select a reviewer.
Assigning a reviewer is mandatory if you set `reviewers.required` to `true` for this transition.

> In order to be able to search for users for review, the user must have policy content/read without limitation or with a limitation that will allow reading users, 

Reviewer: ```suggestion
To be able to search for users for review, the user must have the content/read Policy without any Limitation,
or with a Limitation that allows reading users.
```

> that is, in addition to your own settings for this policy, you must add to limitation subtree /Users and add Users in the content type limitation.

Reviewer: ```suggestion
This means that, in addition to your own settings for this Policy, you must add the /Users subtree to the Limitation and add Users in the [Content Type Limitation](limitation_reference.md#content-type-limitation).
```
